django/river/api/views.py

```diff
@@ -20,15 +23,27 @@ class BatchViewSet(viewsets.ModelViewSet):
     def create(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
+
+        resource_ids = serializer.data["resource_ids"]
```

Comment:
Is there a difference between serializer.data["resource_ids"] and serializer.validated_data["resource_ids"]?

Reply:
https://stackoverflow.com/questions/42000687/what-are-the-differences-between-data-and-validated-data